src/components/Patient/PatientDetailsTab/Demography.tsx

```diff
@@ -188,12 +188,19 @@ export const Demography = (props: PatientProps) => {
           ),
         },
         {
-          label: t("date_of_birth"),
-          value: (
+          label: t(
+            patientData.date_of_birth ? "date_of_birth" : "year_of_birth",
+          ),
+          value: patientData.date_of_birth ? (
             <>
               {dayjs(patientData.date_of_birth).format("DD MMM YYYY")} (
               {formatPatientAge(patientData, true)})
             </>
+          ) : (
+            <>
+              {new Date().getFullYear() - patientAgeInYears(patientData)} (
```

Comment:
backend does provide year of birth which we can use directly

```suggestion
              {patientData.year_of_birth} (
```